src/common/k8s-api/kube-api.ts

```diff
@@ -270,16 +268,16 @@ export class KubeApi<T extends KubeObject> {
    * First tries options.apiBase, then urls in order from options.fallbackApiBases.
    */
   private async getLatestApiPrefixGroup() {
-    // Note that this.options.apiBase is the "full" url, whereas this.apiBase is parsed
-    const apiBases = [this.options.apiBase, ...this.options.fallbackApiBases];
+    const apiBases = [this.options.apiBase || this.objectConstructor.apiBase, ...this.options.fallbackApiBases].filter(Boolean);
 
     for (const apiUrl of apiBases) {
-      // Split e.g. "/apis/extensions/v1beta1/ingresses" to parts
-      const { apiPrefix, apiGroup, apiVersionWithGroup, resource } = parseKubeApi(apiUrl);
-
-      // Request available resources
       try {
-        const response = await this.request.get<IKubeResourceList>(`${apiPrefix}/${apiVersionWithGroup}`);
+        // Split e.g. "/apis/extensions/v1beta1/ingresses" to parts
+        const { apiPrefix, apiGroup, apiVersionWithGroup, resource } = parseKubeApi(apiUrl);
+        const url = `${apiPrefix}/${apiVersionWithGroup}`;
```

Comment:
I can move this change out to a different PR if necessary, but this fixes a bug with fallback as fallback would fail completely if the `apiUrl` is not valid.